index.js

```diff
@@ -3,17 +3,17 @@ const createModuleZero = require('module-zero');
 const m0 = createModuleZero({
     files: '**/*',
     devDependencies: {
-        "eslint": "^6.8.0",
-        "eslint-config-airbnb-base": "^14.0.0",
-        "eslint-config-prettier": "^6.9.0",
-        "eslint-plugin-import": "^2.19.1",
-        "eslint-plugin-jest": "^23.3.0",
-        "eslint-plugin-prettier": "^3.1.2",
-        "fs-extra": "^8.1.0",
-        "husky": "^4.0.3",
-        "jest": "^24.9.0",
-        "lint-staged": "^9.5.0",
-        "prettier": "^1.19.1"
+        "eslint": "^7.25.0",
+        "eslint-config-airbnb-base": "^14.2.1",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-import": "^2.22.1",
+        "eslint-plugin-jest": "^24.3.6",
+        "eslint-plugin-prettier": "^3.4.0",
+        "fs-extra": "^9.1.0",
+        "husky": "^6.0.0",
+        "jest": "^26.6.3",
+        "lint-staged": "^10.5.4",
+        "prettier": "^2.2.1"
```

Comment:
There are quite a few major version bumps... do they all work with the existing configs? https://github.com/CriminalInjuriesCompensationAuthority/q-base-config/tree/master/blocks